kin-sdk/KinAccount.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading.Tasks;
+using kin_base;
+
+namespace kin_sdk
+{
+    public class KinAccount
+    {
+        public readonly KeyPair keyPair; 
+        private readonly KinClient client;
+
+        internal KinAccount(KeyPair keyPair, KinClient client)
+        {
+            this.keyPair = keyPair;
+            this.client = client;
+        }
+
+        public string GetPublicAddress() => this.keyPair.AccountId;
```

Comment:
this can be converted to a property..
 
public string GetPublicAddress => this.keyPair.AccountId;